packages/transition-frontend/webpack.config.js

```diff
@@ -30,13 +30,13 @@ const entryFileName =  './lib/app-transition.js';
 
 module.exports = (env) => {
   console.log(`building js for project ${config.projectShortname}`);
-
-  const isProduction = env === 'production';
   console.log('process.env.NODE_ENV', process.env.NODE_ENV);
 
-  const bundleFileName = isProduction ? `transition-${config.projectShortname}-bundle-${env}.[contenthash].js` : `transition-${config.projectShortname}-bundle-${env}.dev.js`;
+  const isProduction = process.env.NODE_ENV === 'production';
+
+  const bundleFileName = isProduction ? `transition-${config.projectShortname}-bundle-${process.env.NODE_ENV}.[contenthash].js` : `transition-${config.projectShortname}-bundle-${process.env.NODE_ENV}.dev.js`;
   const styleFileName = isProduction ? `transition-${config.projectShortname}-styles.[contenthash].css` : `transition-${config.projectShortname}-styles.dev.css`;
-  const htmlFileName = path.join(`index-${config.projectShortname}${env === 'test' ? `_${env}` : ''}.html`);
+  const htmlFileName = path.join(`index-${config.projectShortname}${process.env.NODE_ENV === 'test' ? `_${process.env.NODE_ENV}` : ''}.html`);
```

Comment:
Can you add a comment to this line to explain what it does ? I want to have a confirmation that the logic is right.